### ISSUE TITLE
Prevent out-of-domain wx model runs

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja#141]
I just accidentally selected NOMADS-HIRES-NMM-AK-5-KM for a TX domain. A wx model file was downloaded and I didn't get an error until I started the simulation. After I started the simulation, I just got "Unhandled exception." 

Can we change this so we check the extents of the wx models like we do for SRTM data and not allow users to choose wx models outside of their bounds? At the least, we should provide some useful info about why the simulation failed.

**Comment by ksshannon** [comment]
Yes, assuming we can gather the grid data and store it in WINDNINJA_DATA.
